SingularityBase/src/main/java/com/hubspot/singularity/SingularityUserFacingAction.java

```diff
@@ -0,0 +1,12 @@
+package com.hubspot.singularity;
+
+public enum SingularityUserFacingAction {
+  RUN_REQUEST,
```

Comment:
naming nits:
- Not sure we need the _REQUEST on each of these. Will likely be easier for users to remember in their configs without it
- Maybe RUN -> EXEC/EXECUTE?

Reply:
Agreed on dropping _REQUEST.

I feel like RUN matches up with the ui ("Run Now"), where EXEC/EXECUTE isn't a term users typically use, so it may be easier to remember.